Makefile

```diff
@@ -172,50 +172,45 @@ clean-bindings:
 	rm -f $(PKG_BINDINGS_DIR)/types_gen.go
 	rm -f $(PKG_BINDINGS_DIR)/zz_generated.api.go
 
-# Update nvml.h from the Anaconda package repository
+# Update nvml.h from the NVIDIA headers repo on gitlab
 update-nvml-h: JQ ?= $(DOCKER) run -i --rm -v "$(PWD):$(PWD)" -w "$(PWD)" backplane/jq:latest
-update-nvml-h: NVML_DEV_PACKAGES_INFO := $(shell \
-		wget -qO - https://api.anaconda.org/package/nvidia/cuda-nvml-dev/files | \
-			$(JQ) '.[] | select(.attrs.subdir=="linux-64") | .version + "@" + .upload_time[:19] + "@" + .full_name' | \
+update-nvml-h: NVML_DEV_HEADERS_INFO := $(shell \
+		wget -qO - https://gitlab.com/api/v4/projects/nvidia%2Fheaders%2Fcuda-individual%2Fnvml_dev/repository/tags | \
```

Comment:
How about using `git clone` rather than querying the RESTful API? Since we are querying a git repo.

```console
$ git clone https://gitlab.com/nvidia/headers/cuda-individual/nvml_dev.git nvml-dev
$ cd nvml-dev
$ git tag --list 'v*' --sort='-version:refname' --format='%(refname:short)%09%(*authordate:format:%y-%m-%d-%H:%M:%S)'
v12.5.39    24-05-15-22:26:21
v12.4.127   24-03-28-23:49:40
v12.4.99    24-02-29-05:54:15
v12.3.101   23-11-09-11:05:04
v12.3.52    23-10-16-04:09:26
v12.2.140   23-08-23-07:12:43
v12.2.128   23-07-27-08:52:32
v12.2.81    23-06-26-10:09:29
v12.1.105   23-04-17-17:34:53
v12.1.55    23-02-23-09:20:23
v12.0.140   23-01-28-19:30:42
v12.0.76    22-12-03-11:07:09
v11.8.86    22-09-29-20:24:19
v11.7.91    22-07-29-12:29:29
v11.7.50    22-05-04-18:22:18
v11.6.55    22-01-11-16:52:51
v11.5.50    22-01-10-17:04:04
v11.4.120   22-01-28-00:02:05

$ git checkout v12.5.39
Note: switching to 'v12.5.39'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 878f50f CUDA 12.5.0 nvML_DEV 12.5.39
$ cp -f nvml.h ../gen/nvml/nvml.h
```